lib/server.js

```diff
@@ -175,7 +183,8 @@ function handleRequest (opts, req, resp) {
             query: urlData.query,
             method: req.method.toLowerCase(),
             headers: req.headers,
-            body: body
+            body: body,
+            reqId: reqId
```

Comment:
Well, headers are not cloned deeply, so child requests shouldn't matter. I was mainly thinking about the logic in the restbase constructor, which currently checks for either reqid *or* the header, on each internal request. If we instead set the header once, we could basically do `this.reqId = par.reqId || req.headers && req.hearders['x-request-id'];`